### DISCUSSION TITLE
Define a media type for annotation sets

### DISCUSSION THREAD

**Discussion by llemeurfr** [readium/annotations#4] [General]
This specification introduces a dedicated media type value to identify an AnnotationSet: application/rd-annotations+json.

I propose using a “rd-” prefix (for ReaDium). It would allow for easier registration at [IANA](https://www.iana.org/assignments/media-types/media-types.xhtml). 

Is it a good idea? 
Should we extend this to other media types, including “application/webpub+json”?

**Comment by HadrienGardeur** [comment]
This feels like a halfway point between media types in the vendor tree (`application/vnd-*`) and standard tree (let's say `application/pdf`).

For any standard that we'd like to push to a standard body recognized by IANA, it's best to target the standard tree. For anything that should remain vendor-specific, it's best to use the vendor tree.

**Reply by llemeurfr** [reply]
This is a generic take. Now look at the IANA registry, there are sets of values prefixed by "alf+", "alto-", 'atsc-", "cdmi-" ... These prefixes represent families.
